src/main/java/org/folio/service/routinglists/validators/RoutingListValidatorUtil.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.service.routinglists.validators;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.folio.rest.core.exceptions.ErrorCodes;
+import org.folio.rest.jaxrs.model.Error;
+import org.folio.rest.jaxrs.model.Location;
+import org.folio.rest.jaxrs.model.PoLine;
+import org.folio.rest.jaxrs.model.RoutingListCollection;
+
+public class RoutingListValidatorUtil {
+
```

Comment:
if it is utility class than private constructor can be created to avoid initialization from outer places. See examples of other utility classes